pages/blog/index.js

```diff
@@ -129,18 +123,30 @@ function BlogIndex({posts}) {
 }
 
 BlogIndex.defaultProps = {
-  posts: null
+  posts: null,
+  pagination: null,
+  tags: null,
+  tagsList: null
 }
 
 BlogIndex.propTypes = {
-  posts: PropTypes.array
+  posts: PropTypes.array,
+  pagination: PropTypes.object,
+  tags: PropTypes.array,
+  tagsList: PropTypes.array
```

Comment:
Les `props` sont toujours définies dans le `getServerSideProps`.
```suggestion
  posts: PropTypes.array.isRequired,
  pagination: PropTypes.object.isRequired,
  tags: PropTypes.array.isRequired,
  tagsList: PropTypes.array.isRequired
```